### PR TITLE
issue: 1520155 Update Sockperf's manual page

### PR DESCRIPTION
**Pull request by liranoz12**
Signed-off-by: Liran Oz <lirano@mellanox.com>

### CODE REVIEW

**Comment by liranoz12** [comment]
bot:retest

File: doc/main.dox [hunk 1]
```diff
@@ -52,18 +52,18 @@
...
    - Provide full log of packet times;
-   - Provide few modes to monitor multiple file descriptors as recv/select/epoll/poll;
+   - Provide few modes to monitor multiple file descriptors as recvfrom/select/poll/epoll;
    - Improved CPU utilization;
    
-   Initially the tool was developed to demonstrate advantages of VMA (Voltaire Messaging Accelerator).
-   VMA is is a multicast-offload, dynamically linked, user space Linux library which serves to 
-   transparently enhance the performance of multicast networking heavy applications over 
-   the InfiniBand and 10 Gb Ethernet network.
+   Initially the tool was developed to demonstrate advantages of VMA (Mellanox Messaging Accelerator).
```
Comment (igor-ivanov) [inline doc/main.dox]: Consider `Mellanox's Messaging Accelerator (VMA) https://github.com/mellanox/libvma/wiki`

**Comment by liranoz12** [comment]
@igor-ivanov - updated

**Review by igor-ivanov — APPROVED** [review]
:+1:

**Pull request merged**